recipe/meta.yaml

```
@@ -1,39 +1,67 @@
 {% set name = "emoji" %}
-{% set version = "2.2.0" %}
+{% set version = "2.12.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: a2986c21e4aba6b9870df40ef487a17be863cb7778dcf1c01e25917b7cd210bb
+  sha256: 4aa0488817691aa58d83764b6c209f8a27c0b3ab3f89d1b8dceca1a62e4973eb
+  patches:
+    - 0001-factor-out-testutils.patch
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
```

Requires python version bound
```suggestion
        skip: True  # [py<37]
```

https://github.com/carpedm20/emoji/blob/v2.12.1/pyproject.toml#L9C1-L9C26